pkgs/sdk_triage_bot/lib/src/common.dart

```diff
@@ -37,9 +37,11 @@ String get geminiKey {
   return token;
 }
 
-/// Don't return more than 4k of text for an issue body.
+/// Don't return more than 5k of text for an issue body.
```

Comment:
(Just curious) Why the limit, if we have a huge context window available?

Reply:
I think this is me being defensive? I can get some data, but I suspect that 90%+ of comments fit in this limit, and the ones that don't are almost always something like a little bit of content and a huge amount of pasted stack trace.

We could keep this as is, raise it substantially so that it really is only protecting against sending in too many tokens, or only have this size limit apply when we're tuning the model (which does have an overall size limit for the tuning data).